db/migrate/20150709221755_create_logins.rb

```diff
@@ -1,4 +1,4 @@
-class CreateLogins < ActiveRecord::Migration
+class CreateLogins < ActiveRecord::Migration[4.2]
```

Comment:
thoughtbot/clearance@b0ce189 uses a generator and includes the `[5.0]` for recent Rails versions. We don't generate the migration file but have actual migrations included in the engine.